layers/+tools/transmission/README.org

```diff
@@ -63,52 +64,68 @@ Add =(transmission :variables transmission-auto-refresh-all t)= to
-| ~SPC m s d~ | Set global download speed limit in kB/s.                      |
-| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point. |
-| ~SPC m g i~ | Open a =transmission-info-mode= buffer for torrent at point.  |
-| ~SPC m s l~ | Set global seed ratio limit.                                  |
-| ~SPC m m~   | Move torrent at point or in region to a new location.         |
-| ~SPC m r~   | Prompt to remove torrent at point or torrents in region.      |
-| ~SPC m t s~ | Toggle torrent between started and stopped.                   |
-| ~SPC m a a~ | Add announce URLs to torrent or torrents.                     |
-| ~SPC m s u~ | Set global upload speed limit in kB/s.                        |
-| ~SPC m v~   | Verify torrent at point or in region.                         |
-| ~SPC m q~   | Quit and bury the buffer.                                     |
-| ~SPC m s b~ | Set bandwidth priority of torrent(s) at point or in region.   |
+| Key binding | Description                                                       |
+|-------------+-------------------------------------------------------------------|
+| ~H~         | Move torrent to the top of the queue.                             |
+| ~J~         | Move torrent down the queue by one.                               |
+| ~K~         | Move torrent up the queue by one.                                 |
+| ~L~         | Move torrent to the bottom of the queue.                          |
+| ~SPC m m~   | Move torrent at point or in region to a new location.             |
+| ~SPC m q~   | Quit and bury the buffer.                                         |
+| ~SPC m v~   | Verify torrent at point or in region.                             |
+| ~SPC m D~   | Prompt to remove and delete torrent at poit or torrens in region. |
+| ~SPC m R~   | Prompt to remove torrent at point or torrents in region.          |
+| ~SPC m S~   | Sort torrents according to the column at point.                   |
+| ~SPC m a a~ | Add torrent by filename, URL, magnet link, or info hash.          |
+| ~SPC m a t~ | Add announce URLs to torrent or torrents.                         |
+| ~SPC m g i~ | Open a =transmission-info-mode= buffer for torrent at point.      |
+| ~SPC m g f~ | Open a =transmission-files-mode= buffer for torrent at point.     |
+| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point.     |
+| ~SPC m s p~ | Set bandwidth priority of torrent(s) at point or in region.       |
+| ~SPC m s d~ | Set global download speed limit in kB/s.                          |
+| ~SPC m s l~ | Set label(s) for marked torrent(s) or torrent at point.           |
+| ~SPC m s r~ | Set global seed ratio limit.                                      |
+| ~SPC m s u~ | Set global upload speed limit in kB/s.                            |
+| ~SPC m t m~ | Toggle mark on torrent(s) at point or in region.                  |
```

Comment:
I think we can make toggle mark `SPC m m`, whose default binding is `m`

Reply:
I had thought of having it in `SPC m t m` because it was a toggle. But if having it out in `SPC m m ` then I believe adding `SPC m x` for `transmission-toggle` (toggle between activating and deactivating a torrent process) should be added too.

Reply:
I have just remembered that ~SPC m m~ to toggle marks would clash with ~transmission-move~.

Reply:
Tying with the comment about fixing the keybindings for ~transmission-remote~ and ~transmission-delete~, we can just have
"R": ~transmission-remove~ for "[R]emove"
"D": ~transmission-delete~ for "[D]elete"
"r": ~transmission-move~ for "[r]ename"
"m": ~transmission-toggle-mark~ for "[m]ark"

Reply:
Made the changes as mentioned in the comment above this.